### PR TITLE
Release v1.1.0

### PR DESCRIPTION
**Pull request by bogdandrutu**
Per the decision we should do a new release of the specs every month to ensure we include all the new changes.

Signed-off-by: Bogdan Drutu <bogdandrutu@gmail.com>

### CODE REVIEW

**Review by cijothomas — APPROVED** [review]

**Review by carlosalberto — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,8 @@ release.
 
 ## Unreleased
 
+## v1.1.0 (2021-03-11)
+
 - Adds note that implementations can ignore links with invalid SpanContext([#1492](https://github.com/open-telemetry/opentelemetry-specification/pull/1492))
```
Comment (yurishkuro) [inline CHANGELOG.md:12]:
```suggestion
- Implementations can ignore links with invalid SpanContext([#1492](https://github.com/open-telemetry/opentelemetry-specification/pull/1492))
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -7,6 +7,8 @@ release.
...
 - Adds `none` as a possible value for OTEL_TRACES_EXPORTER and OTEL_METRICS_EXPORTER to disable export
```
Comment (yurishkuro) [inline CHANGELOG.md:13]:
```suggestion
- Add `none` as a possible value for OTEL_TRACES_EXPORTER and OTEL_METRICS_EXPORTER to disable export
```

**Review by yurishkuro — COMMENTED** [review]
I would suggest that if we want to do regular releases, then the changelog should be better structured, at minimum broken down into subsections for different signals & semantic conventions. Right how all items are in a single pile.

I would also suggest setting higher bar for the PR titles, which translate into changelog entries (imperative tense, etc. Cf. https://chris.beams.io/posts/git-commit/#seven-rules), so that they all read consistently.

**Comment by bogdandrutu** [comment]
@yurishkuro I gave a try to split by signal and semantic conventions

**Review by yurishkuro — APPROVED** [review]
thanks!

**Pull request merged**